src/test/java/com/hubspot/jinjava/lib/exptest/IsWithinExpTestTest.java

```diff
@@ -53,12 +53,6 @@ public void itFailsOnNullValueNotInSequence() {
       .isEqualTo("fail");
   }
 
-  @Test
-  public void itFailsOnNullSequence() {
-    assertThat(jinjava.render(String.format(IN_TEMPLATE, "2", "null"), new HashMap<>()))
-      .isEqualTo("fail");
-  }
-
```

Comment:
This is related to whether we should return false or throw an exception. In Python, `2 in None` would throw an exception.

Reply:
Escaping is tricky. I would leave out the changes to the escape filter and the addition of the escaped expression test in a separate PR to be safe.

Reply:
That makes sense.

Reply:
I removed the escaping, but also this code snippet here is related to the whether searching something that isn't an iterable should return false as it does currently or throw an exception. Can you comment on that?